static/src/javascripts/projects/common/modules/commercial/creatives/add-tracking-pixel.js

```diff
@@ -0,0 +1,17 @@
+define([
+    'common/utils/template',
+    'text!common/views/commercial/tracking-pixel.html'
+], function (
+    template,
+    trackingPixelStr
+) {
+    var trackingPixelTpl;
```

Comment:
Might be a silly question due to lack of context, but why not just `trackingPixelTpl = template(trackingPixelStr);`?